### PR TITLE
Hyc 852 helper tests

### PR DESCRIPTION
**Pull request by smithjp**
* https://jira.lib.unc.edu/browse/HYC-871

### CODE REVIEW

File: spec/helpers/hyrax_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe HyraxHelper do
+  describe '#language_links' do
+    let(:options) { {value: ['http://id.loc.gov/vocabulary/iso639-2/eng']} }
+
+    it 'returns a link to a language search' do
```
Comment (bbpennel) [inline spec/helpers/hyrax_helper_spec.rb]: add test for a KeyError case to ensure it returns nil

File: spec/helpers/hyrax_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      expect(helper.language_links(options)).to eq '<a href="/catalog?f%5Blanguage_sim%5D%5B%5D=http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Fiso639-2%2Feng">English</a>'
+    end
+  end
+
+  describe '#language_links_facets' do
+    let(:options) { 'http://id.loc.gov/vocabulary/iso639-2/eng' }
+
+    it 'returns a link to a language search' do
```
Comment (bbpennel) [inline spec/helpers/hyrax_helper_spec.rb]: add KeyError test
